lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml

```diff
@@ -179,9 +179,21 @@ states:
         targetState: APP_DOC_CHECK_GIVEN_ONLY
         checkFeatureFlag:
           strategicAppEnabled:
-            targetState: STRATEGIC_APP_TRIAGE
+            targetState: STRATEGIC_APP_TRIAGE_GIVEN_ONLY
             targetEntryEvent: appTriage
 
+  STRATEGIC_APP_TRIAGE_GIVEN_ONLY:
+    nestedJourney: STRATEGIC_APP_TRIAGE
+    exitEvents:
+      next:
+        targetState: POST_APP_DOC_CHECK_GIVEN_ONLY
+      sessionError:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+      anotherWay:
+        targetJourney: FAILED
+        targetState: FAILED_UPDATE_DETAILS
+
```

Comment:
Not for this ticket/PR, but I think we have a gap since the introduce of the DL check as part of `APP_DOC_CHECK` which we'll need to bring into strategic app?